examples/multisig/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "local-project-test",
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "chai": "^4.2.0"
+  },
+  "dependencies": {
+    "algob": "^0.0.5",
```

Comment:
if we are using `yarn link algob` then we should remove this line.

Reply:
ok